tests/conftest.py

```diff
@@ -39,6 +39,10 @@
 
 import pytest
 
+import sys
+if "volttron" not in sys.path:
+    sys.path.insert(0, "src/volttron")
```

Comment:
so sys.path is an array...matching on volttron will never match src/volttron.

Also, you should be adding the src directory not the src/volttron directory

Finally if this is run from tests directory it has a different result than running from the root directory.